src/m365/pp/commands/managementapp/managementapp-list.ts

```diff
@@ -0,0 +1,36 @@
+import { Logger } from '../../../../cli';
+import request from '../../../../request';
+import GraphCommand from '../../../base/GraphCommand';
+import commands from '../../commands';
+
+class PpManagementAppListCommand extends GraphCommand {
+  public get name(): string {
+    return commands.MANAGEMENTAPP_LIST;
+  }
+
+  public get description(): string {
+    return 'Lists management applications for Power Platform';
+  }
+
```

Comment:
Since this is a `list` command, we should define the list of default properties to be included in the text and csv formats.

Reply:
@waldekmastykarz the response is always only an array of applicationids: 

```json
"value": [
      {
        "applicationId": "31359c7f-bd7e-475c-86db-fdb8c937548e"
      }
    ]
```

Do we still introduce default properties in that case?

Reply:
We could make it explicit in case the API changes and would make the response unreadable